cairo1-run/src/main.rs

```diff
@@ -320,6 +323,116 @@ fn main() -> Result<(), Error> {
     }
 }
 
+// Serializes the return values in a user-friendly format
+// Displays Arrays using brackets ([]) and Dictionaries using ({})
+// Recursively dereferences referenced values (such as Span & Box)
```

Comment:
If this is intended to be the documentation, then it should be prefixed with the triple slash `///`.

Reply:
Good call!